models/AmazonS3.cfc

```diff
@@ -54,23 +54,30 @@ component accessors="true" singleton {
      */
     public AmazonS3 function init(
         required string accessKey,
-        required string secretKey,
+		required string secretKey,
```

Comment:
Can you add the arguments to the CFC docs please. Also they should be proper properties as well

Reply:
javadoc notation added. 
Explain what you mean about proper properties please.

Reply:
Adding them as `property name="secretKey"`

Reply:
Basically adding them to the pseudo constructor so they are documented with auto getters and setters:

```
property name="awsRegion";
property name="awsDomain";
```